js/store.js

```diff
@@ -298,10 +301,10 @@ const store = {
       if (!this.isVisibleThirdParty(thirdParty)) {
         if (this.onAllowList(origin, target)) {
           // hide third party
-          thirdParty['isVisible'] = false;
+          thirdParty['isVisible'] = 0;
         } else {
           // show third party; it either became visible or is brand new
-          thirdParty['isVisible'] = true;
+          thirdParty['isVisible'] = 1;
           isNewThirdParty = true;
           for (let i = 0; i < thirdParty['firstPartyHostnames'].length; i++) {
             const firstPartyHostname = thirdParty['firstPartyHostnames'][i];
```

Comment:
The existing code does this:
```
for (const key in data) {
  thirdParty[key] = data[key];
}
```

We will need to do smarter things for merging this for keys we don't want to just update. So `requestTime` should likely be stored as: `firstRequestTime` and `latestRequestTime` (adding both now to account for clearing of data would be worthwhile)